### DISCUSSION TITLE
Favorite Emotes Tab

### DISCUSSION THREAD

**Discussion by brunogbrl** [night/betterttv#5994] [Ideas]
**Favorite emotes tab** is a feature present in many instant messaging apps and online communication platforms. Basically, it allows the user to select their favorite emotes and place them in a specific tab to access them more **quickly** and **conveniently**.

This feature is especially useful for people who use a wide **variety of emotes** in their daily conversations. Instead of having to search for the desired emotes every time they need to use them, the user can simply access their favorite emotes tab and choose the emote they want to use.

In addition, the favorite emotes tab also allows the user to customize their communication experience. By choosing the emotes he uses the most and adding them to the favorites tab, he can make his conversations more **fluid** and **dynamic**, avoiding the need to type words to find him.

Another advantage of the favorite emotes tab is that it can be used as a form of **organization**. A lot of people have a hard time finding the emotes they want out of a huge amount of options. By selecting their favorite emotes and placing them in a specific tab, the user can have quick and easy access to their favorite emotes, without having to go through an endless list of options.

[PREVIEW](https://drive.google.com/file/d/1gXD0TvVLRTyzUWJK14tbQZublxaixp3C/view?usp=sharing)

**Comment by night** [comment]
Thanks for the suggestion. This feature actually already exists in the product. You can hold ALT and click on an emote in the menu to add/remove it as a favorite. Favorites always display at the top of the list when available in a stream you're watching.
